recipes/trio-asyncio/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "trio_asyncio" %}
+{% set version = "0.7.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "0b85aece9250c5db73c22f3a55ed5cd00facdf741d7b672935c1ca88e83f9436" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python 3.6.*
+    - pip
+    - pytest-runner
+    - setuptools
+  run:
+    - python 3.6.*
+    - trio
+    - async_generator >=1.6
+    - contextvars >=2.1
+
+test:
+  imports:
+    - trio_asyncio
+
+about:
+  home: https://github.com/python-trio/trio-asyncio
+  license: MIT
+  license_family: MIT
```

Comment:
Could you manually package the `license_file`?

https://conda-forge.org/docs/meta.html#packaging-the-license-manually